### PR TITLE
Make reusable app with unit tests

### PR DESCRIPTION
**Pull request by ZuluPro**

### CODE REVIEW

**Comment by ZuluPro** [comment]
@tkanemoto Any news ?

File: dj_torrent/__init__.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+"""
+Automatically search and download subscribed TV shows and magazines using
+torrentz.eu, myepisodes.com and transmissionrpc
```
Comment (tkanemoto) [inline dj_torrent/__init__.py]: Note to self: remove the bit about myepisodes.com

File: dj_torrent/urls.py [hunk 2]
```diff
@@ -11,15 +11,15 @@
             TemplateView.as_view(template_name="torrent/torrent.html")),
         name='torrent',
     ),
-    url(r'^(?P<id>[0-9]+)$',
+    url(r'^(?P<id>[0-9-]+)$',
```
Comment (tkanemoto) [inline dj_torrent/urls.py]: Do we need the hyphen?

**Comment by tkanemoto** [comment]
Generally excellent, and thanks for this. Just a couple of minor comments.

File: dj_torrent/urls.py (hunk 2 above)
Comment (ZuluPro) [inline dj_torrent/urls.py]: No we don't need this. It was a mistake because of new torrent IDs (-1)
I'll correct it

**Comment by tkanemoto** [comment]
Thanks. You can now install this via pip using -e git+https://github.com/tkanemoto/django-torrent#egg=django-torrent

**Pull request merged**